tests/test_grid.py

```diff
@@ -482,14 +482,14 @@ def test_quadrature_grid(self):
         N = 0
         NFP = 1
         grid_quad = QuadratureGrid(L, M, N, NFP)
-        roots, weights = special.js_roots(3, 2, 2)
+        roots, weights = special.js_roots(2, 2, 2)
```

Comment:
Not relevant to this PR but a comment above 485 explaining the purpose of js_roots would be super helpful!

Reply:
A comment is probably overkill when the doc string of special says they are shifted jacobi roots imo, but I can add